docs/patrol/new-native-setup.mdx

```diff
@@ -0,0 +1,310 @@
+---
+title: Patrol - setup
+---
+
+# Native automation setup
+
+Using Patrol's native automation feature requires more setup work than using
+custom finders. Unfortunately it's necessary, because you have to integrate your
+app with Android and/or iOS native testing tools.
+
+We believe that the powerful features you'll get will make up for it!
+
+<Warning>
+        In this tutorial, we are using example app, which has package name
+        `com.example.myapp` on Android, bundle id `com.example.MyApp` on iOS
+        and `My App` name on both Android and iOS. Replace any occurences
+        of those names with proper values.
+</Warning>
+
```

Comment:
During reading this docs for sure I won't remember what I need to swap, we need also to highlight what to replace next to according code snippets

Reply:
It's kinda self explanatory, that if there is something called so poorly like `MyApp`, it should be replaced, I was going to remove this warning entirely